lib/screens/v3_api.ex

```diff
@@ -5,7 +5,7 @@ defmodule Screens.V3Api do
 
   @default_opts [timeout: 2000, recv_timeout: 2000, hackney: [pool: :api_v3_pool]]
 
-  def get_json(route, params \\ %{}, extra_headers \\ [], opts \\ []) do
+  def get_json(route, params \\ %{}, extra_headers \\ [], opts \\ [], include_headers \\ false) do
```

Comment:
Hmm, I think a term like `include_headers` usually flags whether or not the request is passed the headers, but here it's used to flag whether the returned data should have the header.  What do you think of calling it `return_with_headers`?

Reply:
That's a good idea. Especially not that we are dealing with request and response headers now.